.github/workflows/simulation-1.1.yml

```diff
@@ -323,13 +323,14 @@ jobs:
       run: |
         cd /tmp/ot-commissioner
         script/bootstrap.sh
-        cmake -GNinja                           \
-              -DCMAKE_CXX_STANDARD=11           \
-              -DCMAKE_CXX_STANDARD_REQUIRED=ON  \
-              -DCMAKE_BUILD_TYPE=Release        \
-              -DCMAKE_INSTALL_PREFIX=/usr/local \
-              -DOT_COMM_COVERAGE=ON             \
-              -DOT_COMM_CCM=OFF                 \
+        cmake -GNinja                            \
+              -DCMAKE_POLICY_VERSION_MINIMUM=3.5 \
```

Comment:
Is it better to update `googletest`?

https://github.com/google/googletest/blob/52204f78f94d7512df1f0f3bea1d47437a2c3a58/CMakeLists.txt#L4

Reply:
Submitting https://github.com/openthread/ot-commissioner/pull/314

Reply:
But there are more ot-commissioner third parties which still specify min cmake version lower than 3.5, e.g. https://github.com/jimsch/cn-cbor/blob/master/CMakeLists.txt. So looks like we still need the PR here